docs/source/markdown/podman-import.1.md

```diff
@@ -34,6 +34,18 @@ Set commit message for imported image
 
 Shows progress on the import
 
+#### **--os**
```

Comment:
Please put these in alphabetical order in the manpage

Reply:
Similar valid values question.  If it take any string such as "sillyOS", then we should state that too.

Reply:
@TomSweeneyRedHat User is free to set any value if he is doing it manually however it's user's responsibility to make it work if they choose a wrong value.

Reply:
Probably worth a follow-up PR since os, arch, variant are mentioned numerous times in the man pages.  Would be nice to have them consistent.